sdk/core/azure-core/test/ut/string.cpp

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: MIT
 
 #include "gtest/gtest.h"
-#include <azure/core/azure.hpp>
+#include <azure/core/strings.hpp>
```

Comment:
Do we have this header? 
string util functions are in Azure.hpp
```cpp
bool LocaleInvariantCaseInsensitiveEqual(const std::string& lhs, const std::string& rhs) noexcept;
  std::string const ToLower(const std::string& src) noexcept;
  unsigned char ToLower(const unsigned char src) noexcept;
```

Reply:
Reverted this change for now.  I'm doing the azure.hpp cleanup in another change